code/modules/mob/living/brain/brain_item.dm

```diff
@@ -180,7 +180,7 @@
 ////////////////////////////////////TRAUMAS////////////////////////////////////////
 
 /obj/item/organ/brain/proc/has_trauma_type(brain_trauma_type, resilience = TRAUMA_RESILIENCE_ABSOLUTE)
-	for(var/X in traumas)
+	for(var/X in shuffle(traumas))
```

Comment:
Seems bit unnecessary for every has_trauma call. I feel adding a helper that returns all traumas of given type would be better.

Reply:
Or instead of returning as soon as it finds the type, make a list of all the correct traumas found and return a pick(found_traumas). Although it is less fast.